packages/moralis/src/UAuthMoralisConnector.ts

```diff
@@ -0,0 +1,127 @@
+import type UAuth from '@uauth/js'
+import type {UAuthConstructorOptions, UserInfo} from '@uauth/js'
+
+import AbstractWeb3Connector from './AbstractWeb3Connector'
```

Comment:
I'm pretty sure that the `AbstractWeb3Connector` class is exported from the moralis library, even though there aren't types for it. I think we should use the imported version instead of redeclaring the class.